reactor-core/src/main/java/reactor/core/publisher/Operators.java

```diff
@@ -1097,6 +1097,21 @@ public static void reportSubscriptionSet() {
 		return new ScalarSubscription<>(subscriber, value);
 	}
 
+	/**
```

Comment:
can you please reuse the wording I suggested in my earlier comment? 🙇

Reply:
still missing the little sentence in the body (line 1102)

Reply:
done